src/rag/vector_store/pgvector.py

```
@@ -1,28 +1,391 @@
+        """커넥션 풀에서 커넥션을 가져옴."""
+        pool = self._get_pool()
+        with pool.connection() as conn:
+            # pgvector extension 등록 (필수)
+            try:
+                from pgvector.psycopg import register_vector
+                register_vector(conn)
+            except ImportError as e:
+                raise RuntimeError(
+                    "pgvector package is required for embedding column operations. "
+                    "Install with: pip install pgvector"
+                ) from e
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)